### ISSUE TITLE
Configuration for rule "space-before-function-paren" is invalid

### ISSUE THREAD

**Issue by rsancheza09** [eslint/eslint#7925]
**Tell us about your environment**

* **ESLint Version:** 3.13.1
* **Node Version:** 6.3.1
* **npm Version:** 3.10.3

**What parser (default, Babel-ESLint, etc.) are you using?**
Babel-ESLint

**Please show your full configuration:**

```js
{
  "parser": "babel-eslint",
  "extends": ["eslint:recommended"],
  "parserOptions": {
    "ecmaFeatures": {
      "jsx": true,
      "modules": true
    }
  },
  "settings": {
    "react": {
      "pragma": "React",
      "version": "0.14.7"
    }
  },
  "env": {
    "browser": true,
    "node": true,
    "mocha": true,
    "es6": true
  },
  "rules": {
    "space-before-function-paren": [2, { "anonymous": "always", "named": "always", "asyncArrow": "ignore" }]
  }
}
```
**What did you expect to happen?**

No issues

**What actually happened? Please include the actual, raw output from ESLint.**

I am getting this error:

_**Configuration for rule "space-before-function-paren" is invalid: Value "[object Object]" no (or more than one) schemas match.**_

**Note:** ATOM IDE

**Comment by platinumazure** [comment]
Do you get this error when running ESLint on the command line?

As far as I can tell, the configuration is valid. My best guess is your Atom/ESLint integration is using an older version of ESLint where that configuration is invalid.

**Comment by rsancheza09** [comment]
I am using this Atom Package https://github.com/AtomLinter/linter-eslint
```
"dependencies": {
    "atom-linter": "^8.0.0",
    "atom-package-deps": "^4.0.1",
    "consistent-path": "^2.0.1",
    "escape-html": "^1.0.3",
    "eslint": "^3.6.0",
    "eslint-rule-documentation": "^1.0.0",
    "process-communication": "^1.1.0",
    "resolve-env": "^1.0.0"
  },
  "devDependencies": {
    "babel-cli": "^6.18.0",
    "babel-preset-node5": "^12.0.0",
    "eslint": "^3.12.0",
    "eslint-config-airbnb-base": "^11.0.0",
    "eslint-plugin-import": "^2.2.0",
    "rimraf": "^2.5.4"
  },
```

**Comment by mysticatea** [comment]
To make sure:

- Did you restart your editor after you upgrade ESLint?
- What version does `eslint --version` command show?
- What version does `./node_modules/.bin/eslint --version` command show?

**Comment by eslintbot** [comment]
Hi @rsancheza09, thanks for the issue. It looks like there's not enough information for us to know how to help you.

If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

Requesting a rule change? Please see [Proposing a Rule Change](http://eslint.org/docs/developer-guide/contributing/rule-changes) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by alberto** [comment]
Closing this issue since we haven't heard anything back for quite some time. Feel free to create a new issue if this is something you would like to discuss further.
